src/engine.c

```diff
@@ -2329,6 +2334,14 @@ void phase_explore(game *g)
 			/* Skip powers that do not have extra draw */
 			if (!(o_ptr->code & P1_DRAW)) continue;
 
+			/* Add value special case : draw per rebel military */
+			if (o_ptr->code & P1_PER_REBEL_MILITARY)
+			{
+				draw += o_ptr->value * count_active_flags(g, i,
+						(FLAG_REBEL | FLAG_MILITARY));
```

Comment:
Use spaces instead of tabs after you have aligned with the previous line. (In total 4 tabs and 8 spaces.)